dev/ci/user-overlays/08893-herbelin-master+moving-evars-of-term-on-econstr.sh

```diff
@@ -0,0 +1,11 @@
+if [ "$CI_PULL_REQUEST" = "8893" ] || [ "$CI_BRANCH" = "master+moving-evars-of-term-on-econstr" ]; then
+
+    Equations_CI_BRANCH=master+fix-evars_of_term-pr8893
+    Equations_CI_REF=master+fix-evars_of_term-pr8893
+    Equations_CI_GITURL=https://github.com/herbelin/Coq-Equations
+
+    equations_CI_BRANCH=master+fix-evars_of_term-pr8893
+    equations_CI_REF=master+fix-evars_of_term-pr8893
+    equations_CI_GITURL=https://github.com/herbelin/Coq-Equations
+
+fi
```

Comment:
Please list the overlay PR in the Coq PR's first message
Also you only need the 2nd triple (there is no `Equations_CI_FOO` variable)

Reply:
Done